### DISCUSSION TITLE
Elementor widgets not loading (keeps loading forever)

### DISCUSSION THREAD

**Discussion by ROZABOY** [elementor/elementor#16600] [Developers Community]
i am a web designer and for the past 2 weeks elementor kept all my time setting up so many pages from the beginning on new installations
with elementor widgets are not loading for ever!!!!

i will definitely get back to visual composer for it's reliable and easy to copy the codes on any not working situations

but mean while since i don't want to stay on running new WordPress installation and all the hustle of exporting and importing to get the same version with the same set of theme and plugins work again, please some one offer some solution other than what can already be found on internet, because believe me a have tried every single one of them including:
1- system requirements
2- new update
3- rolling back to older versions
4- memory limits
5- adding actual memory to my hosting plan
6- adding more memory just becuase
7- disabling all the themes & plugins except elementor it self
8- using Hello & Twenty One 
9- editing wp-config.php + .htaccess + PHP.ini 
10- disabling the preloader
11- activating the safe mode / 
12- adding this line to wp-config: if ($_SERVER['HTTP_X_FORWARDED_PROTO'] == 'https') $_SERVER['HTTPS']='on';
13- disabling browser addons
14- playing every scenario with Elementor settings
15- adding this to htacces:
<IfModule mod_substitute.c>
SubstituteMaxLineLength 30m
</IfModule>
LimitRequestBody 9999999

16- cleaning up the browser on every damn changes
17- PHP z lib extension is enable on my server
18- installing browsers that are not chrome
19- updating my php version to 8
20- so many other stuff

oooohhhh, how much i hate this right now, what have i done, why did i switch at the first place, so much time wasted so many website still are not able to change or edit, i hate this mess

**worst part is i copy the same files and database to a subdomain, some export & import process, and it will work !?!?!? there is no different !?!? why does it work**

**Comment by aaliyahfraser** [comment]
You can use Elemntor addons. Elementor has so many free amazing add-ons. Easy-to-use features made creating websites a breeze. There are available free & premium widgets that are really well-designed & load fastly. Here we talk about “**[Best Elementor Addons and Plugins for WordPress 2023 – Compared](https://quomodosoft.com/best-elementor-addons-and-plugins/)**” You can check out it now1
